### PR TITLE
Changed the logout button order

### LINKED ISSUE #153: Change the order of the Logout with the admin / company section in the UserMenu
It makes more sense to have the Logout button at the end of the user menu.
You can make the necessary modifications to the code [here](https://github.com/NIAEFEUP/nijobs-fe/blob/develop/src/components/Navbar/UserMenu.js#L147).

image

### PR DESCRIPTION
**Pull request by dfneves19**
Changed the logout button order so it appears last.
closes #153

### CODE REVIEW

**Review by TiagooGomess — APPROVED** [review]

**Comment by TiagooGomess** [comment]
You should rebase with develop before merging

File: src/components/Navbar/UserMenu.js
Comment (TiagooGomess) [inline src/components/Navbar/UserMenu.js:154]:
```suggestion
            </Typography>
            {sessionData?.company && <Divider className={classes.divider} />}
```

Comment (TiagooGomess) [reply src/components/Navbar/UserMenu.js:154]: I'm sorry for changing my review; I only saw this after approving the PR.

**Review by TiagooGomess — CHANGES_REQUESTED** [review]

**Comment by DoStini** [comment]
The divider is not yet fixed isn't it? @dfneves19 

image

File: src/components/Navbar/UserMenu.js
Comment (DoStini) [inline src/components/Navbar/UserMenu.js:162]: In order to fix the divider not being the same width as the top one, I think you need to remove this `classes.divider`. @TiagooGomess

**Review by DoStini — CHANGES_REQUESTED** [review]

**Comment by dfneves19** [comment]
while logged in as a company the dividers are exactly the same, but when logged in as admin the logout divider is still a little deviated to the left.

**Comment by CarlosMealha** [comment]
So I fixed the divider problem with a trivial solution, adding a break statement just to know if it's an admin or a company account authenticated, and then I duplicated the style, just changing the width percentage.  What do you guys think?

**Comment by Naapperas** [comment]
@DoStini please review this PR (since you requested changes previously, also because I would appreciate a second opinion)

**Comment by CarlosMealha** [comment]
> @DoStini please review this PR (since you requested changes previously, also because I would appreciate a second opinion)

I think this would fix the problem
image

**Review by Naapperas — APPROVED** [review]
Rebase

**Comment by BrunoRosendo** [comment]
@CarlosMealha This was already approved but it needs to be rebased. Shouldn't take too long

**Comment by CarlosMealha** [comment]
> @CarlosMealha This was already approved but it needs to be rebased. Shouldn't take too long

Okay, it's already done, please aprove it.

**Pull request merged**
